src/displayapp/DisplayApp.cpp

```diff
@@ -382,6 +391,37 @@ void DisplayApp::Refresh() {
           LoadNewScreen(Apps::Alarm, DisplayApp::FullRefreshDirections::None);
         }
         break;
+      case Messages::WakeAlarmTriggered:
+        if (currentApp == Apps::Sleep) {
+          auto* sleep = static_cast<Screens::Sleep*>(currentScreen.get());
+          sleep->SetAlerting();
+        } else {
+          LoadNewScreen(Apps::Sleep, DisplayApp::FullRefreshDirections::None);
+        }
+        break;
+      case Messages::GradualWakeTriggered:
+        if (currentApp == Apps::Sleep) {
+          // auto* sleep = static_cast<Screens::Sleep*>(currentScreen.get());
```

Comment:
The if else here is completely empty due to commented out code. I assume this is unintentional and not the final state.